eslint.config.js

```diff
@@ -45,6 +45,7 @@ export default [
 			'sites/svelte-5-preview/**',
 			// wasn't checked previously, reenable at some point
 			'sites/svelte.dev/**',
+			'playgrounds/demo/src/**',
```

Comment:
You've added this to “wasn't checked previously, reenable at some point” section. Shouldn't it be added above that comment?

Reply:
ah good catch. it's really only the one line that the comment should apply to (i.e. not `tmp/**`) — reordered in https://github.com/sveltejs/svelte/commit/50a9020d00e04a1e31f02c0e76eb21371e14d85f